### PR TITLE
Add AbortController timeout to the fetch method

### PR DESCRIPTION
**Pull request by simonvbrae**
Added a timeout as discussed.
Not sure what to do about the formatting.

### CODE REVIEW

**Comment by rubensworks** [comment]
It looks like your editor auto-applied some formatting, could you undo those?

**Comment by rubensworks** [comment]
We may also need some (or at least one) unit tests for this new functionality.

**Comment by rubensworks** [comment]
It looks like the error message is different on Node 16. Perhaps we should just use `toThrow()` without arguments (less clean, but we'll have to live with it).

File: lib/SparqlEndpointFetcher.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export class SparqlEndpointFetcher {
   public readonly sparqlParsers: {[contentType: string]: ISparqlResultsParser};
   public readonly sparqlJsonParser: SparqlJsonParser;
   public readonly sparqlXmlParser: SparqlXmlParser;
+  public readonly timeout: number;
+  public readonly defaultTimeout: number = 5000;
```
[the remark this answers is not in the document]
Comment (simonvbrae) [reply lib/SparqlEndpointFetcher.ts]: Done!

**Comment by rubensworks** [comment]
Great work @simonvbrae, thanks for this!
Released as 3.2.0.

Just removing your node_modules and lockfile + re-installing should get you this new version.

**Pull request merged**